app/services/record_declaration.rb

```diff
@@ -0,0 +1,151 @@
+# frozen_string_literal: true
+
+class RecordDeclaration
```

Comment:
So it likes this PR only covers the endpoint for recording declarations. Will there be further PRs to cover off the other endpoints such as defer/resume and the classes that power those?

Reply:
Yes there were the stories for the rests of class.